sql/updates/latest-dev.sql

```diff
@@ -0,0 +1,18 @@
+CREATE OR REPLACE FUNCTION ts_realtime_agg_update() returns void LANGUAGE PLPGSQL AS
+$BODY$
+DECLARE
+ vname text;
+ altercmd text;
+BEGIN
+    FOR vname IN select view_name::text from timescaledb_information.continuous_aggregates where materialized_only = false
+    LOOP
+        altercmd := 'ALTER VIEW ' || vname || ' SET (timescaledb.materialized_only=false) ' ; 
```

Comment:
this is not safe and you should use format or quote_ident to escape the identifier name

Reply:
modified the function

Reply:
Had to remove the specific format call. It turns out that the cast from regclass to text already applies the formatting. When doing this twice we end up with something like 
``ALTER VIEW """matMm2""" SET (timescaledb.materialized_only=false) ``